src/remote/client/interface.cpp

```diff
@@ -995,6 +998,18 @@ class RProvider : public StdPlugin<IProviderImpl<RProvider, CheckStatusWrapper>
 void RProvider::shutdown(CheckStatusWrapper* status, unsigned int /*timeout*/, const int /*reason*/)
 {
 	status->init();
+
+	// make OS abort all outgoing connections
+	MutexLockGuard g(outPortsMutex, FB_FUNCTION);
+	for (auto p = outPorts->begin(); p != outPorts->end(); ++p)
```

Comment:
Why not `for (auto port : outPorts)`?

Reply:
It does not work with GlobalPtr<> variable.